proposals/NNNN-DisabledSoftbuttons.md

```diff
@@ -0,0 +1,89 @@
+
+# Disabled Softbuttons
+
+* Proposal: [SDL-NNNN](NNNN-DisabledSoftbuttons.md)
+* Author: [Michael Crimando](https://github.com/MichaelCrimando)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / Web / RPC ]
```

Comment:
Sorry, I missed this in my first review: please specify what `Web` refers to here - Policy Server and/or SHAID?

Reply:
Ah whoops didn't mean to leave that in